.travis.yml

```diff
@@ -8,6 +8,8 @@ sudo: false
 before_install:
    - export PATH=$HOME/gopath/bin:$PATH
 
+install: "pip install --user -r requirements.txt"
```

Comment:
also, this line `"pip install ..."` should be moved in the `before_install` block.